### PR TITLE
 [ALLUXIO-2068] Fix Javadoc style for {{@param}} in UnderFileSystem.java

### PR DESCRIPTION
**Pull request by HongweiQin**
https://alluxio.atlassian.net/browse/ALLUXIO-2068
Apart from fixing words like "The",  I also change those words like "UFS" to "ufs".
Besides, I also add some "the" to precede the noun.
Hope it's right.

### CODE REVIEW

File: core/common/src/main/java/alluxio/underfs/UnderFileSystem.java [hunk 1]
```diff
@@ -418,8 +418,8 @@ public abstract OutputStream create(String path, CreateOptions options)
   /**
    * Gets the list of locations of the indicated path given its offset.
    *
-   * @param path The file name
-   * @param offset Offset in bytes
+   * @param path the file name
+   * @param offset offset in bytes
```
Comment (HorizonNet) [inline core/common/src/main/java/alluxio/underfs/UnderFileSystem.java]: To be consistent it should be "the offset in bytes".

File: core/common/src/main/java/alluxio/underfs/UnderFileSystem.java [hunk 2]
```diff
@@ -591,8 +591,8 @@ public abstract boolean mkdirs(String path, MkdirsOptions options)
    * The default implementation simply concatenates the path to the base URI. This should be
    * overridden if a subclass needs alternate functionality.
    *
-   * @param ufsBaseUri the base {@link AlluxioURI} in the UFS
-   * @param alluxioPath the path in Alluxio from the given base
+   * @param ufsBaseUri the base {@link AlluxioURI} in the ufs
+   * @param alluxioPath the path in alluxio from the given base
```
Comment (HorizonNet) [inline core/common/src/main/java/alluxio/underfs/UnderFileSystem.java]: I think "Alluxio" should still start with an upper-case letter.

**Comment by HorizonNet** [comment]
@HongweiQin Thanks for the contribution. I left two minor comments. The rest looks good to me.

**Comment by HongweiQin** [comment]
@HorizonNet Thanks for your comment. I've changed those two parts and made a new commit.

**Comment by apc999** [comment]
thanks @HongweiQin for submitting the patch and improve the javadoc
thanks @HorizonNet for reviewing the PR.
LGTM

**Pull request merged**